src/state/slices/txHistorySlice/utils.ts

```diff
@@ -47,3 +53,24 @@ export const serializeTxIndex = (
 
   return [accountId, txid, address.toLowerCase()].join(UNIQUE_TX_ID_DELIMITER)
 }
+
+export const deserializeTxIndex = (txIndex: TxIndex): TxDescriptor => {
```

Comment:
Unused in the end, but this guy will definitely come handy so leaving it here unless we decided it's scope creep